packages/mattermost-redux/src/actions/errors.ts

```diff
@@ -69,7 +67,6 @@ export function logError(error: ServerError, displayable = false, consoleError =
             serializedError.message = 'A JavaScript error has occurred. Please use the JavaScript console to capture and report the error';
         }
 
-        EventEmitter.emit(ErrorTypes.LOG_ERROR, error);
```

Comment:
This event is actually left over from the mobile app. The web app doesn't use this event at all

Reply:
If that's the case, should we also remove the action and reducer?

```
packages/mattermost-redux/src/actions/errors.ts
30:        type: ErrorTypes.LOG_ERROR,

packages/mattermost-redux/src/reducers/errors/index.ts
13:    case ErrorTypes.LOG_ERROR: {
```

Reply:
Those are still fine. The redux part still controls the "A JavaScript error has occured" announcement bar. We just previously reused that constant as the name of an event that the mobile app needed for something similar

Reply:
Ahh, ok. :+1: